themes/frontierline/inc/customizer.php

```diff
@@ -87,27 +87,44 @@ function frontierline_customize_register($wp_customize) {
 
   $wp_customize->add_control('frontierline_firefox_button', array(
     'label'       => esc_html__('Firefox download button', 'frontierline'),
-    'description' => esc_html__('Display a "Download Firefox" button in the main navigation in place of the "Discover Firefox" link.', 'frontierline'),
+    'description' => esc_html__('Display a “Download Firefox” button in the main navigation in place of the “Discover Firefox” link.', 'frontierline'),
     'priority' => 7,
     'section'     => 'frontierline_theme_options',
     'settings'    => 'frontierline_firefox_button',
     'type'        => 'checkbox',
   ));
 
+  // Register hero image option and control
+  $wp_customize->add_setting('frontierline_no_post_image', array(
+    'capability'        => 'edit_theme_options',
+    'default'           => '',
+    'sanitize_callback' => 'frontierline_sanitize_checkbox',
+    'type'              => 'theme_mod',
+  ));
+
+  $wp_customize->add_control('frontierline_no_post_image', array(
+    'label'       => esc_html__('Disable featured images on posts', 'frontierline'),
```

Comment:
Disable featured images on post *pages*. 

Since they're still set on the post even when disabled.